PL2/PL2-shared/src/main/java/nl/tudelft/pl2016gr2/model/OriginalGraph.java

```diff
@@ -17,10 +17,14 @@ public OriginalGraph() {
   /**
    * Print a string representation of this graph.
    */
-  public void print() {
+  @Override
+  public String toString() {
+    StringBuilder sb = new StringBuilder();
     for (Node node : nodes.values()) {
-      System.out.println(node);
+      sb.append(node);
+      sb.append("\n");
```

Comment:
Appending '\n' as character is more efficient (not really important for a toString, but usually StringBuilder is used for efficiency, so just some advice).
Also in my opinion chaining the calls looks to stringbuilder nicer:
sb.append(node).append('\n');